bin/test_e2e

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash -e
+
+#
+# Remote Integration Test Runner
+#
+# This runs only e2e features, which are those dependent on
+# access to a remote PCF instance. Run this script from the `bin`
+# directory with `summon ./test_e2e`, with
+# proper summon credentials.
+#
+
+# First, check for summon variables
+: ${CF_API_ENDPOINT?"Error: Need to set CF_API_ENDPOINT"}
```

Comment:
Needs double quotes. I *think* nested quotes will work here:
```
: "${CF_API_ENDPOINT?"Error: Need to set CF_API_ENDPOINT"}"
```
or maybe use single quotes in the interior.